mmhuman3d/core/visualization/visualize_smpl.py

```diff
@@ -163,32 +163,41 @@ def _prepare_background(image_array, frame_list, origin_frames, output_path,
     return image_array, remove_folder, frames_folder
 
 
-def _prepare_body_model(model_type, body_model, model_path, gender):
-    """Prepare `body_model` from `model_path` or existing `body_model`."""
+def _prepare_body_model(model_type, body_model, body_model_config):
+    """Prepare `body_model` from `body_model_config` or existing
+    `body_model`."""
     if model_type not in ['smpl', 'smplx']:
         raise ValueError(
             f'Do not support {model_type}, please choose in `smpl` or `smplx.')
     if body_model is None:
-        if model_path is not None:
-            if osp.isdir(model_path):
+        if body_model_config is not None:
+            model_path = body_model_config.get('model_path', None)
+
+            use_pca = body_model_config.get('use_pca', False)
+            body_model_config.update(use_pca=use_pca)
+
+            model_type = body_model_config.get('type', model_type)
+            body_model_config.update(type=model_type.lower())
+
+            use_face_contour = body_model_config.get('use_face_contour', True)
+            body_model_config.update(use_face_contour=use_face_contour)
+
+            num_betas = body_model_config.get('num_betas', 10)
+            body_model_config.update(num_betas=num_betas)
+
```

Comment:
Line 174-186, I think we do not have to provide default values, because all default values will be provided by smplx.

Reply:
> Line 174-186, I think we do not have to provide default values, because all default values will be provided by smplx.

The default value of `use_face_contours` is `False` thus the output points is 127, not 144. Not very compatible with our settings. How to solve this?

Reply:
I don't think we should add default values in the visualization modules. This is quite similar to the `flat_hand` discussion. I think there are two solutions:
1. The users explicitly indicate `use_face_contours=True` when passing the config.
2. We change the default values in SMPL wrapper.

Reply:
By now, solution 2 is not possible in today, maybe I should change all the default values in test

Reply:
I changed this in test

Reply:
I agree, in test, we can provide a config with all values that we want.